.expeditor/config.yml

```diff
@@ -4,3 +4,8 @@ project:
 
 github:
   delete_branch_on_merge: true
+
+pipelines:
+  - verify: 
```

Comment:
What behavior would you like this pipeline to exhibit on master itself? The way it works by default is to just execute the tests, but with your code as presented it would likely be a "no-op". There's no setting for it in the config.yml, but we may want to disable master testing all together to save on confusion.

Reply:
I think we should disable testing on master for now. There aren't currently any actions that needs to be taken once a PR is merged.